simple-example/simple-example.cabal

```diff
@@ -37,8 +37,9 @@ test-suite doctests
   main-is:          doctests.hs
   build-depends:
       base
-    , doctest           >=0.15 && <0.17
-    , QuickCheck        >=2.12 && <2.14
+    , base-compat       >=0.11.0 && <0.12
```

Comment:
let's have `0.10.5` as a lower bound, as it's the version in [the current lts](https://www.stackage.org/lts-14.7)